pointcloud/build.sbt

```diff
@@ -23,3 +23,5 @@ fork in Test := true
 parallelExecution in Test := false
 
 javaOptions += s"-Djava.library.path=${Environment.ldLibraryPath}"
+
+testOptions in Test += Tests.Argument("-oDF")
```

Comment:
Probably we need to throw it away; as it prints complete stack traces (potentially not good for travis)